scripts/test.sh

```diff
@@ -13,7 +13,7 @@ cleanup() {
   fi
 }
 
-ganache_port=8545
+ganache_port=8111
```

Comment:
since we are here, I'd like to ask why we have to kill previous ganache instance?
Isn't keeping it faster for running local tests?
This question has bugged me for a long time but I always forgot to ask :laughing:

Reply:
You nailed it 😆 . It's a relict from old times. We probably should stop killing it.